workflows/metabolomics/qcxms-sdf/QCxMS-Spectra-Prediction-from-SDF-tests.yml

```diff
@@ -0,0 +1,13 @@
+- doc: Test outline for QCxMS-Spectra-Prediction-from-SDF
+  job:
+    Input SDF File:
+      class: File
+      path: test-data/Input SDF File.sdf
+      filetype: sdf
+  outputs:
+    Predicted Spectra:
+      path: test-data/Predicted Spectra.msp
```

Comment:
If you include the path a line-by-line comparison is run. You can  skip this (and delete the file) if you only want to assert the number of line.
```suggestion
```